psycop/projects/sequence_models/fine_tune_w_config.py

```diff
@@ -0,0 +1,54 @@
+import datetime as dt
+from pathlib import Path
+
+from psycop.common.feature_generation.loaders.raw.load_ids import SplitName
+from psycop.common.feature_generation.sequences.cohort_definer_to_prediction_times import (
+    CohortToPredictionTimes,
+)
+from psycop.common.feature_generation.sequences.patient_loaders import (
+    DiagnosisLoader,
+    PatientLoader,
+)
+from psycop.common.sequence_models import train
+from psycop.common.sequence_models.dataset import PatientSlicesWithLabels
+from psycop.common.sequence_models.registry import Registry
+from psycop.common.sequence_models.tasks import BEHRTForMaskedLM
+from psycop.projects.t2d.feature_generation.cohort_definition.t2d_cohort_definer import (
+    T2DCohortDefiner,
+)
+
+
+@Registry.datasets.register("model_from_checkpoint")
+def load_model_from_checkpoint(
+    checkpoint_path: Path,
+) -> BEHRTForMaskedLM:
+    return BEHRTForMaskedLM.load_from_checkpoint(checkpoint_path)
+
+
+@Registry.datasets.register("patient_slices_with_labels_for_t2d")
```

Comment:
Registered twice?

Reply:
The other one is a tmp file as we haven't redone the pretrain / fine-tune. I would add them, like we see here (specifically for the use case).